rdkit/Chem/UnitTestCatalog.py

```diff
@@ -158,7 +157,7 @@ def _test5MoreComplex(self):
     for i, mol in enumerate(suppl):
       fp = fpgen.GetFPForMol(mol, self.fragCat)
       for bit in ranges[i]:
-        assert fp[bit], '%s: %s' % (Chem.MolToSmiles(mol), str(bit))
+        assert fp[bit], f'{Chem.MolToSmiles(mol)}: {str(bit)}'
```

Comment:
No need for `str(bit)`

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.